packages/app/src/widgets/position-section/position-section.tsx

```diff
@@ -12,13 +15,48 @@ import { SupportedToken } from '@/shared/stellar/constants/tokens'
 import { StakeDecreaseModal } from '@/features/borrow-flow/components/stake-decrease-modal'
 import { getStakeUsd, getDebtUsd, sumObj } from './utils'
 
+const sorobanTokenRecordToPositionCell = (tokenRecord: SorobanTokenRecord): PositionCell => ({
+  value: Number(tokenRecord.balance) / 10 ** tokenRecord.decimals,
+  type: tokenRecord.symbol.substring(1).toLowerCase() as PositionCell["type"],
+})
+
 export function PositionSection() {
+  const userAddress = useContextSelector(WalletContext, (state) => state.address)
   const position = useContextSelector(PositionContext, (state) => state.position)
   const setPosition = useContextSelector(PositionContext, (state) => state.setPosition)
   const [decreaseDebtModalIndex, setDecreaseDebtModalIndex] = useState<0 | 1 | null>(null)
   const [increaseDebtModalIndex, setIncreaseDebtModalIndex] = useState<0 | 1 | null>(null)
   const [decreaseStakeModalIndex, setDecreaseStakeModalIndex] = useState<0 | 1 | null>(null)
 
+  const debtXlmBalance = useGetBalance(tokens.xlm.debtAddress, userAddress)
+  const lendXlmBalance = useGetBalance(tokens.xlm.sAddress, userAddress)
+
+  const debtXrpBalance = useGetBalance(tokens.xrp.debtAddress, userAddress)
+  const lendXrpBalance = useGetBalance(tokens.xrp.sAddress, userAddress)
+
+  const debtUdcBalance = useGetBalance(tokens.usdc.debtAddress, userAddress)
+  const lendUdcBalance = useGetBalance(tokens.usdc.sAddress, userAddress)
```

Comment:
Let's rewrite this to SUPPORTED_TOKENS mapping. SUPPORTED_TOKENS defines order of tokens display. It'll also reduce repetitions.

Reply:
useGetBalance refactored for work with arrays